packages/drng/ticker.go

```diff
@@ -120,8 +132,9 @@ func (t *Ticker) send() {
 				}
 			case <-t.dRNGTicker.C:
 				// still no new randomness within awaitOffset, take the default value, and reset dRNGTicker
+				delay := time.Since(start)
+				t.setDelayedRoundStart(delay)
```

Comment:
this is not correct, because the ticker is set for 7 second. so the delay in the DelayedRoundStart should be precisely 3sec